.rubocop.yml

```diff
@@ -5,3 +5,7 @@ inherit_gem:
 inherit_mode:
   merge:
     - Exclude
+
+Metrics/BlockLength:
+  Exclude:
+    - db/schema.rb
```

Comment:
If you also enable Rails cops, `db/schema.rb` will be excluded automatically. But that's probably way more work.